pycbc/events/coherent.py

```diff
@@ -80,7 +80,9 @@ def get_coinc_triggers(snrs, idx, t_delay_idx):
     coincs: dict
         Dictionary of coincident trigger SNRs in each detector
     """
-    coincs = {ifo: snrs[ifo][idx + t_delay_idx[ifo]] for ifo in snrs}
+    coincs = {
+        ifo: snrs[ifo][(idx + t_delay_idx[ifo]) % len(snrs[ifo])]
```

Comment:
I'm not quite sure what `snrs[ifo]` is actually storing at this point .... or is the problem that idx + t_delay_idx[ifo] might wrap around the start (or the end)? We do want to keep track of this, so that any quoted delay times in the output file correspond to physical shift (so if we are at t=1, and shift by -100, we give the time-slide as len - 100, not as -100.)